src/app

```diff
@@ -20,15 +20,8 @@ FILES="$2"
 
 temp_dir="$FILES"/tmp
 
-cleanup() {
-    set +e
-    test -f "$PERSISTENT_STORE"/.rw_test && rm -f "$PERSISTENT_STORE"/.rw_test
-}
-
-trap cleanup 1 2 3 6 15
-
```

Comment:
out of curiosity: why are we removing the poor old trap?

Reply:
Since the file is immediately removed I found it easier to just remove it and disable the `-e` flag. Also, the file is never cleaned up on the exit condition (`0`).